tests/python/contrib/test_arm_compute_lib/test_network.py

```diff
@@ -151,7 +152,34 @@ def get_model():
     )
 
 
+# Test disabled as it requires attontate target not promote unannotated tuples under previous operations annotation
```

Comment:
```suggestion
# Test disabled as it requires annotate target not promote unannotated tuples under previous operations' annotation
```
If this test cannot be executed in this PR, we should add it along with the one it can be.

Reply:
Test removed for PR it belongs